api/repositories.py

```diff
@@ -117,7 +117,7 @@ def validate_token(self, house_id, token):
         if house is None:
             return False
         else:
-            user_id = house['user_id']
+            user_id = house.get_house_attributes()['user_id']
```

Comment:
This would work, but why don't you use house.user_id?
Definitely not a big deal, but could make the code more robust (automatic refactoring etc.)